src/core/wee-config-file.c

```diff
@@ -1196,11 +1161,8 @@ config_file_option_reset (struct t_config_option *option, int run_callback)
                     || (strcmp ((char *)option->value,
                                 (char *)option->default_value) != 0))
                     rc = WEECHAT_CONFIG_OPTION_SET_OK_CHANGED;
-                if (option->value)
-                {
-                    free (option->value);
-                    option->value = NULL;
-                }
+                FREE(option->value);
+                option->value = NULL;
```

Comment:
[R1165](https://github.com/weechat/weechat/pull/866/files#diff-3ba66fdfee8c62012bd7f1fe08767041R1165) and [R1200](https://github.com/weechat/weechat/pull/866/files#diff-3ba66fdfee8c62012bd7f1fe08767041R1200) also removable?